Dockerfile

```diff
@@ -1,15 +1,21 @@
+# Build our upstream version of atlantis while this is being developed
+FROM golang:1.13 as builder
+WORKDIR /go/src/github.com/runatlantis/atlantis
+COPY . /go/src/github.com/runatlantis/atlantis
+RUN GO111MODULE=off CGO_ENABLED=0 go build -o atlantis main.go
```

Comment:
Do we not want to be basing on 1.14 since that is what is in the repo and tests now?
With 1.14 as well I presume that we may want to be using go modules?